src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -272,39 +278,14 @@ private void GetProjectItems(out IList<NavigationBarProjectItem> projectItems, o
                     d.Project.GetGlyph(),
                     workspace: d.Project.Solution.Workspace,
                     documentId: d.Id,
-                    language: d.Project.Language)).OrderBy(projectItem => projectItem.Text).ToList();
-
-            projectItems.Do(i => i.InitializeTrackingSpans(_subjectBuffer.CurrentSnapshot));
```

Comment:
the call to initialize tracking spans can be removed because that is done during model computation?  And that must have happened for whatever we have currently?

Reply:
No.  In this case we can remove it because project items have no tracking spans :)  they don't correspond to any code.  The list of spans/tracking-spans is always empty for them no matter what.